apps/ledger-live-mobile/src/screens/PairDevices/index.tsx

```diff
@@ -43,6 +43,8 @@ type PairDevicesProps = Props & {
 };
 type RouteParams = {
   onDone?: (_: Device) => void;
+  /** If defined, only show devices that have a device model id in this array */
+  deviceModelIds?: DeviceModelId[];
```

Comment:
But so why did you made this change ? Are you using this for the CLS ? Do you want to directly use the new BLE pairing flow (especially if your feature will stay behind a feature flag and won't be released right now): `apps/ledger-live-mobile/src/screens/BleDevicePairingFlow/index.tsx`